mods/dungeon_loot/loot.lua

```diff
@@ -1,26 +1,10 @@
 dungeon_loot.registered_loot = {
```

Comment:
would be useful to add a comment above the table that explains that only items from `default` are registered here and the rest in the respective mods

Reply:
Done; I wrote the following;

-- Loot from the `default` mod is registered here,
-- with the rest being registered in the respective mods